### PR TITLE
CLI flag to disable installer CLI telemetry

### PR DESCRIPTION
**Pull request by malnick**
Satisfies https://mesosphere.atlassian.net/browse/DCOS-9283

### CODE REVIEW

**Comment by malnick** [comment]
@spahl This is ready for review

File: ext/dcos-installer/dcos_installer/cli.py [hunk 1]
```diff
@@ -50,7 +50,15 @@ def print_usage():
 Install Mesosophere's Data Center Operating System
 
 dcos_installer [-h] [-f LOG_FILE] [--hash-password HASH_PASSWORD] [-v]
```
Comment (cmaloney) [inline ext/dcos-installer/dcos_installer/cli.py]: `[--cli-telemetry-disabled]` should go here based on how the argument is added (it's not part of the mutually exclusive set).

File: ext/dcos-installer/dcos_installer/cli.py [hunk 2]
```diff
@@ -50,7 +50,15 @@ def print_usage():
 Install Mesosophere's Data Center Operating System
```
Comment (cmaloney) [inline ext/dcos-installer/dcos_installer/cli.py]: Not for this PR, but why can't we just use the standard / default generated argparse help nowadays?

Comment (malnick) [inline ext/dcos-installer/dcos_installer/cli.py]: I think we can, we removed it initially to support overriding the PORT env from the docker wrapper (at least that's how I recall the decision to write this in manually, but don't quote me on it).

**Comment by cmaloney** [comment]
🚢

**Pull request merged**